stonesoup/models/measurement/linear.py

```diff
@@ -95,3 +96,71 @@ def covar(self, **kwargs):
         """
 
         return self.noise_covar
+
+class GeneralLinearGaussian(MeasurementModel, LinearModel, GaussianModel):
```

Comment:
Docstring required

Reply:
Hi @sdhiscocks, I wonder if instead of adding this 'GeneralLinearGaussian' into Stone Soup, we could instead update the original 'LinearGaussian' class to include the bias term? Would it be of any interest, or you'd rather have this odd extra class added?

Reply:
Re-written 'GeneralLinearGaussian' so that it now inherits from the original 'LinearGaussian' and generalises it to the case that permits explicitly specifying a measurement matrix (and a bias vector). Also covered in the commit's description.